### ISSUE TITLE
Centos support

### ISSUE THREAD

**Issue by armiller** [svanzoest-cookbooks/camo#4]
I would like to use this cookbook to deploy camo on a Centos 6 server. Almost everything works on Centos but the dependency on start-stop-daemon is a problem. I can open a pull request for Centos 7 support easily (using systemd) but Centos 6 will be more difficult. I was thinking of pulling in the runit cookbook for that. What are your thoughts?

**Comment by svanzoest** [comment]
@armiller any more thoughts on this? I agree that getting centos support would be great.

**Comment by armiller** [comment]
@svanzoest I apologize for the long wait. I completed support for centos 6 and deployed it into production successfully a couple months ago but got side tracked with some other tasks. Before I deployed I was trying to add chef spec tests and ran into some issues and ended up moving on.

I can open a pull request if you would like to see the code. It might also be worth doing a separate pull request to update the chef spec tests. Serverspec tests would also be a nice addition.
